tools/lint/lint.py

```diff
@@ -521,6 +526,9 @@ def check_parsed(repo_root, path, f):
         if timeout_value != "long":
             errors.append(rules.InvalidTimeout.error(path, (timeout_value,)))
 
+    required_elements = []  # type: List[Text]
+
+    testharnessreport_nodes = []  # type: List[ElementTree.Element]
```

Comment:
Should we move `testharnessreport_nodes` and `testdriver_vendor_nodes` into sourcefile to be like `source_file.testharness_nodes` etc?